ClientApp/viking-vault/src/components/styles.css

```diff
@@ -33,12 +29,14 @@
     vertical-align: middle;
 }
 
-.regular-div{
+.dashboard-div{
```

Comment:
dashboard-div isn`t specific enough. More exactly, that -div has no real purpose.